test/t8_geometry/t8_geometry_implementations/t8_gtest_geometry_lagrange.cxx

```diff
@@ -250,3 +249,65 @@ INSTANTIATE_TEST_SUITE_P (t8_gtest_geometry_lagrange, LagrangeCmesh,
               << "_degree" << std::get<1> (info.param);
     return test_name.str ();});
 /* clang-format on */
+
+#if T8_ENABLE_DEBUG
```

Comment:
why is this test only run in debug modus? Maybe use T8_ENABLE_LESS_TESTS instead?

Reply:
Because this check is only run in debug. It is the same as with the negative volumes, the check is rather expensive.